app/build.gradle.kts

```diff
@@ -98,6 +98,7 @@ configurations {
 
 dependencies {
     implementation("com.wire.kalium:kalium-logic")
+    implementation("com.wire.kalium:kalium-util")
```

Comment:
I have a question here 🤔 ... Do we really need to depend on `:util` as well ? I don't see a big problem, except that now we need to be careful and treat `util` as a public facing module. 

So that will imply mark undesired public classes as internal, ie: `KaliumDispatcher`, `FileUtil` ? This is KMP class, `DelicateKaliumApi`  and newer additions there. So that will also require updating the documentation on confluence about Reloaded and depedencies.

Reply:
I think going forward with the new dependency, probably needs a sync on the Collective :D what do you think ?

Reply:
sure i will raise this in collective

Reply:
i needed the util module for the time class